src/ethereum/curve/contracts.ts

```diff
@@ -1077,6 +1083,4780 @@ const contracts = {
       tokens: ["0x3Ed3B47Dd13EC9a98b44e6204A523E766B225811"],
     },
   },
+  CurveMetapoolDepositPool: {
```

Comment:
If there are pools for deposits and swaps, please split them as such. I am unaware what pools we need for Curve, however all of them should be captured in this file with the appropriate naming for identification.